.env.example

```diff
@@ -0,0 +1,5 @@
+ETHERSCAN_API_KEY=
+ALCHEMY_API_KEY=
```

Comment:
if we are cleaning this up we can prob just accept an API key and then update the URL to take the api key and hardcode the rest of the URL